CONTRIBUTING.md

```diff
@@ -17,17 +17,17 @@ You'll need a basic understanding of [Git and GitHub.com](https://guides.github.
 
 ## Markdown syntax
 
-Articles are written in [DocFx-flavored Markdown](http://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html), which is a superset of [GitHub-flavored Markdown (GFM)](https://guides.github.com/features/mastering-markdown/). For examples of DFM syntax for UI features commonly used in the EF documentation, see [Metadata and Markdown Template](https://github.com/dotnet/docs/blob/master/styleguide/template.md) in the .NET Core repo style guide. 
+Articles are written in [DocFx-flavored Markdown](http://dotnet.github.io/docfx/spec/docfx_flavored_markdown.html), a superset of [GitHub-flavored Markdown (GFM)](https://guides.github.com/features/mastering-markdown/). For examples of DFM syntax for UI features commonly used in the EF documentation, see [Metadata and Markdown Template](https://github.com/dotnet/docs/blob/master/styleguide/template.md) in the .NET Core repo style guide. 
 
 ## Folder structure conventions
 
-Images, and other static content, are stored in an `_static` folder within each area/folder of the site.
+Images, and other static contents are stored in an `_static` folder within each area/folder of the site.
```

Comment:
Here we had an incorrect use of Oxford comma with only two elements.

Reply:
I think the concept of "static content" takes "content" in its uncountable form.